### ISSUE TITLE
[v6] external props in Field will not be updated when props changed

### ISSUE THREAD

**Issue by leesiongchan** [erikras/redux-form#936]
```javascript
<Field
  component={celebrity => (
    <span>
      Celebrity: {celebrity.value.name}
      External: {celebrity.externalProp}
    </span>
  )}
  name={name}
  externalProp={this.state.externalProp}
/>
```

`externalProp` changes will not be updated in `Field`.

**Comment by erikras** [comment]
Good catch. 👍

**Comment by erikras** [comment]
Published fix as [`v6.0.0-alpha.9`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.9).
